code/game/machinery/doors/airlock.dm

```diff
@@ -193,10 +193,10 @@ About the new airlock wires panel:
 	return wires.IsIndexCut(wireIndex)
 
 /obj/machinery/door/airlock/proc/canAIControl()
-	return ((aiControlDisabled!=1) && (!isAllPowerLoss()))
+	return ((aiControlDisabled != 1) && (!isAllPowerLoss()))
```

Comment:
What is `1` in this context? TRUE? Another value that should be a define? The numbers are too magical